reference/debugging.md

```diff
@@ -0,0 +1,300 @@
+    ```
+
+3. Next, create an Ambassador Service and deploy a basic `httpbin` Ambassador Mapping by saving the following YAML to a file named `ambassador-services.yaml`
+
+    ```yaml
+    ---
+    apiVersion: v1
+    kind: Service
+    metadata:
+    name: ambassador
+    spec:
+    type: LoadBalancer
+    ports:
+    - port: 80
+    selector:
+    service: ambassador
+    ---
+    apiVersion: getambassador.io/v1
+    kind:  Mapping
+    metadata:
+    name:  httpbin
+    spec:
+    prefix: /httpbin/
+    service: httpbin.org
+    host_rewrite: httpbin.org
+    ```
+
+4. Apply this into your cluster with the following command:
+
+    ```shell
+    $ kubectl apply -f ambassador-services.yaml
+    ```
+
+You should now be able to utilize this example mapping for debugging demonstrations and purposes.
+
+## Check Ambassador Status
+
+First, check to see if the [Diagnostics](/reference/diagnostics) service is reachable with the command: `kubectl port-forward <ambassador_pod_name> 8877`
+
+If it is successful, try to diagnose your original issue with the Diagnostics Console.
+
+**If it is not successful, complete the following to see if Ambassador is running:**
+
+1. Check the Ambassador deployment with the following: `kubectl get deployments`
+2. After a brief period, the terminal will print something similar to the following:
+
+    ```shell
+    kubectl get deployments
+    NAME         DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+    ambassador   3         3         3            3           1m
+    ```
+
+3. Check that the “desired” number of Pods equals the “current” or “available” number of pods. If they are **not** equal, check the status of the associated pods with the following command: `kubectl get pods`
+
+    The terminal should print something similar to the following:
+
+    ```shell
+    $ kubectl get pods
+    NAME                         READY     STATUS    RESTARTS   AGE
+    ambassador-85c4cf67b-4pfj2   1/1       Running   0          1m
+    ambassador-85c4cf67b-fqp9g   1/1       Running   0          1m
+    ambassador-85c4cf67b-vg6p5   1/1       Running   0          1m
+    ```
+
+4. If any of the Pods have a status of “not started,” use the following command to “describe” the Deployment pods: `kubectl describe deployment ambassador`
+
+    * Look for data in the “Replicas” field near the top of the output. For example: 
+        `Replicas: 3 desired | 3 updated | 3 total | 3 available | 0 unavailable`
+
+    * Look for data in the “Events” log field near the bottom of the output, which often displays data such as a fail image pull, RBAC issues, or a lack of cluster resources. For example:
+        ```shell
+        Events:
+        Type    Reason         Age   From                   Message
+        ----    ------         ----  ----                   -------
+        Normal  ScalingReplicaSet  2m    deployment-controller  Scaled up replica set ambassador-85c4cf67b to 3
+        ```
```

Comment:
(1) I don't think we should be doing any syntax-highlighting on this
(2) Why do the columns not line up?